File: Demo/Sources/ViewControllers/Main/MainViewController.m
//
//  Globus
//

#import "MainViewController.h"

@interface MainViewController ()
@end

@implementation MainViewController

#pragma mark - Init / Free

- (void)setup {
    [super setup];
    
    self.allowsDoneBarButton = NO;
    self.toolbarHidden = YES;
}

#pragma mark - UIViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    if(self.URL == nil) {
        self.URL = [NSURL URLWithString:@"http://globus-ltd.ru"];
    }
}

#pragma mark - Actions

- (IBAction)pressedMenu:(id)sender {
    [self.glb_slideViewController showLeftViewControllerAnimated:YES complete:nil];
}

@end

File: Demo/Sources/ViewControllers/Main/MainViewController.swift
//
//  Globus
//

import Globus

class MainViewController: GLBWebViewController {
    
    // MARK - Init / Free
    
    override func setup() {
        super.setup()
    
        self.allowsDoneBarButton = false;
        self.isToolbarHidden = true;
    }
    
    // MARK - UIViewController
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        if self.url == nil {
            self.url = URL.init(string: "http://globus-ltd.ru")
        }
    }
    
    // MARK - Action
    
    @IBAction internal func pressedMenu(_ sender: Any) {
        self.glb_slideViewController?.showLeftViewController(animated: true, complete: nil)
    }
    
    // MARK: - GLBNibExtension
    
    public override static func nibName() -> String {
        return "MainViewController-Swift"
    }
    
}
